python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -841,9 +842,40 @@ async def close_m1_cover(self) -> None:
                 f"{MTMount.DeployableMotionState.DEPLOYED!r}"
             )
 
-    async def home_dome(self) -> None:
-        # TODO: Implement (DM-21336).
-        raise NotImplementedError("# TODO: Implement (DM-21336).")
+    async def home_dome(self, physical_az: float = 0.0) -> None:
+        self.log.info("Homing dome")
+        reported_az = await self.rem.mtdome.tel_azimuth.aget(timeout=self.fast_timeout)
+
+        offset = physical_az - reported_az.positionActual
+        self.log.debug(f"Dome azimuth offset: {offset} degrees")
+        target_az = Angle(self.home_dome_az - offset, unit=u.deg).deg
+        self.rem.mtdome.evt_azMotion.flush()
+        await self.rem.mtdome.cmd_moveAz.set_start(
```

Comment:
The MTCS class has a `slew_dome_to` method that will handle all this for you.

Reply:
You're absolutely right! It slipped through my mind even though I think I wrote that one.

Reply:
Let's use the default 120s for now.